engine/overworld/trainer_sight.asm

```diff
@@ -221,8 +221,8 @@ TrainerEngage:
 	ld a, [wTrainerSpriteOffset]
 	and a
 	jr z, .noEngage
-	ld hl, wFlags_0xcd60
-	set 0, [hl]
+	ld hl, wStrengthFlags
+	set BIT_SEEN_BY_TRAINER, [hl]
```

Comment:
This is an example of why I'm not a fan of naming something based on _part_ of its usage.

Unless _all_ the bits relate to strength, I don't think `wStrengthFlags` is a good name. It leads to head-scratching code like this. _(oops, did someone use the wrong `BIT_*` constant on this line? oh wait, no, it's right..)_

(aside: the gen 2 repos do this a bit more than the gen 1 repos. For example, song files in gen 2 are named after _some_ map that uses the song (even though other maps use that song too), whereas as gen 1 uses cities1.asm, routes1.asm, duneon1.asm, etc, which I strongly prefer.)

Reply:
In principle I agree with you. It's why I went with generic Status names for most of these. I copied the `wStrengthFlags` name from gen 2, which also has some other not-entirely-accurate names like `wBikeFlags`.

Do you have a different name suggestion for this variable, considering it's not part of the Status group?

Reply:
I'm not really bothered by the fact that it isn't located in close proximity to the other status bytes in ram. `wStatusFlagsX` would be fine imo.

Reply:
I really don't think it should share a name with this cluster of distant bytes. `wStrengthFlags` is $CD60, the status flags start at $D728.

Would `wMiscFlags` be too generic?

Reply:
I think that's fine too.

Reply:
In my original PR I considered defining multiple labels for the same address.. and just using the appropriate label in conjunction with the appropriate bit.